basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -93,6 +96,18 @@ public void before() throws IOException, TimeoutException, InterruptedException
         Console.withBrowser(browser).waitUntilLoaded();
     }
 
+    private void waitUntilRunning() throws TimeoutException {
+        String command = "/host=master/server=server-one:read-attribute(name=server-state)";
```

Comment:
Should I add a static String variable with server name then? (In this case, only server-one is important for navigation.) Or should I add more general method to e.g. BasePage? (Which will become obsolete with [creaper release](https://github.com/Creaper-Project/creaper))